app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+
+    sep_command = command.split()
+    if len(sep_command) != 3:
+        raise ValueError
+    com, file_name, new_file_name = sep_command
+
+    update_file_name = ""
+    if os.path.exists(file_name) and com == "mv":
+        if os.path.isdir(new_file_name):
+            directory = new_file_name
+            update_file_name = os.path.join(new_file_name, file_name)
+        else:
+            directory = os.path.dirname(new_file_name)
+            update_file_name = new_file_name
+
+        if not os.path.exists(new_file_name) and directory:
+            os.makedirs(directory, exist_ok=True)
+
+    os.rename(file_name, update_file_name)
```

Comment:
Why do you rename file even if the command not equal to mv? It would be better to move this code to the "if os.path.exists(file_name) and com == "mv":" statement